config/routes.rb

```diff
@@ -3144,6 +3145,7 @@
     ems_network
     ems_physical_infra
     ems_physical_infra_dashboard
+    ems_storage
```

Comment:
Does the `ems_block_storage` route have an `index` action?

Reply:
I think not

Reply:
So please add that ones as well

Reply:
I look into it and ems_block_storage does have an index action when you adding new block storage manager. (I get the following error without it : "No route matches {:action=>"show", :controller=>"ems_block_storage"}, missing required keys: [:id] [ems_block_storage/new]".